src/metadata.cc

```diff
@@ -81,6 +81,22 @@ class MetadataWorker : public Nan::AsyncWorker {
         memcpy(baton->icc, icc, iccLength);
         baton->iccLength = iccLength;
       }
+      // IPTC
+      if (image.get_typeof(VIPS_META_IPCT_NAME) == VIPS_TYPE_BLOB) {
```

Comment:
@jcupitt Was the use of IPCT (vs IPTC) in the `VIPS_META_IPCT_NAME` naming intentional/historical?

Reply:
Oh dear, no, just acronymical dyslexia, sorry. I'll fix it.

Reply:
Fixed in 8.6 with https://github.com/jcupitt/libvips/commit/77287a938f571aed71b403d88f741ef2f097df86

There's a #define so the wrong name will still work.

Reply:
@jcupitt Thanks John, fast work as always.

@oaleynik This PR can use the current IPCT spelling for now. The forthcoming version of sharp will depend on libvips v8.6.1+ so I'll update to use the IPTC naming before release.

Reply:
Rodger! Thanks for the quick fix @jcupitt ;)